src/dataviews/dataview-model-base.js

```diff
@@ -173,10 +176,22 @@ module.exports = Model.extend({
 
   fetch: function (opts) {
     opts = opts || {};
+
     this.trigger('loading', this);
+
+    if (opts.success) {
+      var successCallback = opts && opts.success;
+    }
+
     return Model.prototype.fetch.call(this, _.extend(opts, {
-      error: function () {
-        this.trigger('error');
+      success: function () {
+        successCallback && successCallback(arguments);
+        this.trigger('loaded', this);
```

Comment:
I forgot to mention that we are using this "custom" event due to we need any event to listen in Deep-insights elements. Before we used to use "sync" event, but it doesn't work pretty good because sync doesn't mean the request has been done/finished.